src/components/Card.js

```diff
@@ -44,9 +62,15 @@ export default class Card {
 
   _handleLikeBtn() {
     this._likeButton.classList.toggle("card__heart_active");
+    this._isLiked = !this._isLiked;
```

Comment:
You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.